### PR TITLE
feat: monorepo

### PR DESCRIPTION
**Pull request by mikitabut**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,61 +1,57 @@
...
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/stylistic"
       ],
-      "extends": ["plugin:testing-library/react"],
       "rules": {
-        "testing-library/await-async-queries": "error",
-        "testing-library/no-await-sync-queries": "error",
-        "testing-library/no-debugging-utils": "warn",
-        "testing-library/no-dom-import": "off"
+        // TODO: Check all typescript rules work
```
Comment (mikitabut) [inline .eslintrc.json]: Remove TODO

**Review by IlyaBondar — COMMENTED** [review]
@mikitabut , what's about to add `CODEOWNERS` file where `apps/chat-e2e` folder will be responsibility of @irinakartun . And by default(`*`) owners are developers of chat team

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status:

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status:

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status:

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by nepalevov — DISMISSED** [review]

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by mikitabut** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/442983)

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by mikitabut** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/443007)

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by akryvasheyeu — DISMISSED** [review]

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by mikitabut** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-616.nightly-test.deltixhub.io
>E2E tests status: success

**Review by akryvasheyeu — APPROVED** [review]

**Pull request merged**